src/sc/sidechain.cpp

```diff
@@ -713,20 +713,27 @@ int CSidechain::getNumBlocksForScFeeCheck()
 
 int CSidechain::getMaxSizeOfScFeesContainers()
 {
-    int epochLength = fixedParams.withdrawalEpochLength;
-    int numBlocks = getNumBlocksForScFeeCheck();
-
-    //assert(epochLength > 0);
-    if (epochLength == 0) {
-        maxSizeOfScFeesContainers = 1;
-    } else {
-        maxSizeOfScFeesContainers = numBlocks / epochLength;
-        if (maxSizeOfScFeesContainers == 0 || (numBlocks % epochLength != 0) )
-        {
-            maxSizeOfScFeesContainers++;
+    if (maxSizeOfScFeesContainers == -1) {
+        if (!isNonCeasing()) {
```

Comment:
What about moving getNumBlocksForScFeeCheck outside the if-else, being used in both branches?

```
const int numBlocks = getNumBlocksForScFeeCheck();
if (!isNonCeasing()) {
...
} else {
...
}
```

Reply:
Yes, seems reasonable. Added in c8b2941bc959c29cd0db69eb8df66aa425bccda8